client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -193,11 +194,28 @@ export default function WPCheckout( {
 
 		const loginUrl = login( { redirectTo, emailAddress } );
 
+		recordTracksEvent( 'calypso_checkout_wpcom_email_exists', {
```

Comment:
This is probably fine, but I wonder if there's an advantage to using the redux version of this function? I don't really have any preference myself, but I recently discovered that there's two versions ([and fixed the docs](https://github.com/Automattic/wp-calypso/pull/53526)); it seems that the redux version is more used in calypso and the docs recommend it. That said, I don't personally mind either way.

Reply:
I believe that in some instances, the payload of the redux action is augmented with props such as the current blog id, site type, etc. I'd say it's best to use the redux version.

Reply:
TIL! Yes, it looks like the redux version would be best. Thanks both of you!